clients/submission-client/package.json

```diff
@@ -24,7 +24,8 @@
     "test": "jest",
     "typescript": "tsc",
     "bundle-definition": "webpack",
-    "openapi": "../../scripts/update-openapi.sh https://docs.api.epilot.io/submission-api.yaml",
+    "openapi": "node ../../scripts/update-openapi.js https://docs.api.epilot.io/submission-api.yaml",
+    "openapi:local": "node ../../scripts/update-openapi.js https://docs.api.epilot.io/submission-api.yaml ../../../submission-api/lambda/ApiHandlerFunction/openapi.yml",
```

Comment:
```suggestion
```
I think this one was left here by accident 😎

Reply:
mama mia, true